src/webrtc/groupCall.ts

```diff
@@ -276,6 +279,11 @@ export class GroupCall extends TypedEventEmitter<
         this.emit(GroupCallStatsReportEvent.ByteSentStats, { report });
     };
 
+    private onSummaryStats = (report: SummaryStatsReport): void => {
+        // @TODO: Implement data argumentation
```

Comment:
Augmentation perhaps? Although I'm still not 100% sure what this would mean - maybe a sentence to explain?

Reply:
I'll will remove this comment. 
 
By that I mean that we could attach additional user information to the stats here. We could also intercept the mute events here and add them to the stats.